server/@types/activitiesAPI/index.d.ts

```diff
@@ -3578,7 +3578,7 @@ export interface components {
        * @description The tier code for this activity, as defined by the Future Prison Regime team
        * @example TIER_1
        */
-      tierCode?: string
```

Comment:
Pre-emptive strike from changes for [SAA-1585](https://dsdmoj.atlassian.net/browse/SAA-1585)